subcommand/common/common_test.go

```diff
@@ -46,13 +46,7 @@ func TestConsulLogin(t *testing.T) {
 	tokenFile := WriteTempFile(t, "")
 
 	client := startMockServer(t, &counter)
-	err := ConsulLogin(
-		client,
-		bearerTokenFile,
-		testAuthMethod,
-		tokenFile,
-		testPodMeta,
-	)
+	err := ConsulLogin(client, bearerTokenFile, testAuthMethod, tokenFile, "", testPodMeta)
```

Comment:
we're not testing it here because it's tested in the command where we have test agent?

Reply:
nevermind! I saw that we need to add tests to the command. Not sure if we need tests here since it'll be just changing the mock server. wdyt?

Reply:
Yup!! The reason i did not add a test here was because it was test server and it will be tested by the test testing the command. Good catch!